content/en/Platform Deep Dive/Domains/_index.md

```diff
@@ -65,3 +65,46 @@ You can Search for Hosts, Filter by Status and Filter by date for when the Host
 Filtering for the date can be helpful e.g. if you want to surface only Host that have been discovered within the last week.
 
 ![Search](/deepdive/search.png "Search")
+
+## How does the Takeover Risk work
+
+The Takeover Risk feature considers 5 conditions, that when are true could indicate a potential risk of DNS- or Subdomain takeover. 
+
+1. This host refers to a website or api which did not respond successfully.
+
+What: This host refers to a website or api by a CNAME DNS record which did not respond successfully.
+Why: That could mean you refer to an account or resource which does not exist anymore and could be registered by someone else.
+Action: Check whether the referred website or api still exists and responds as intended.
+Severiy: Low
```

Comment:
The way this is rendered isn't readable as intended.
![image](https://github.com/cobalthq/cobalt-product-public-docs/assets/106162427/06bb8d1b-0c49-4b64-8d2a-ec5ab07c8332)
 I think adding blank lines should improve it
```suggestion
What: This host refers to a website or api by a CNAME DNS record which did not respond successfully.

Why: That could mean you refer to an account or resource which does not exist anymore and could be registered by someone else.

Action: Check whether the referred website or api still exists and responds as intended.

Severiy: Low
```